packages/design/tailwind/css/components.css

```diff
@@ -686,11 +686,23 @@ input[type='file' i] {
 }
 
 .gi-modal-body {
-  @apply gi-px-4 gi-flex-1 xs:gi-px-6;
+  @apply gi-px-6 gi-flex-1 gi-max-h-[280px] xs:gi-max-h-[344px] gi-overflow-y-auto;
```

Comment:
Can we use percentage instead of pixels?

Reply:
keeping min/max (px) make them consistent otherwise they will look weird if the user is changing screen sizing.